### PR TITLE
WF-64957-Documentation-for-find-and-replace-in-Presentation

### PR DESCRIPTION
**Pull request by AkashArul26**
<html>
<body>

Task Link | https://syncfusion.atlassian.net/browse/WF-64957
-- | --
Description | Add UG document for Presentation find and replace
Solution | Added UG document for Presentation find and replace
Content review confirmation | No

</body>
</html>

### CODE REVIEW

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 1]
```diff
@@ -0,0 +1,937 @@
...
+.NET PowerPoint library allows you to replace a specific text in the PowerPoint presentation with another text.
+
+* You can replace either the first occurrence or all the occurrences of the text with another text in all PowerPoint slides and elements such as Shape, Textbox, Table, SmartArt, etc.
+* You can replace the text in specific slide, notes slide, master slide or layout slide.
+* You can replace the text by matching case, whole word and all occurrence or first occurrence alone.
+* You can also replace the content that spans across several lines.
+
+## Find and replace in PowerPoint presentation
+
+You can find text in a Presentation document and replace it with other text.
```
[the remark this answers is not in the document]
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said @Ramaraj-Marimuthu sir

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 2]
```diff
@@ -0,0 +1,937 @@
...
+
+The following code example illustrates how to find a particular text and replace with other text in a Presentation document.
```
[the remark this answers is not in the document]
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said @Ramaraj-Marimuthu sir

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 3]
```diff
@@ -0,0 +1,937 @@
...
+	Xamarin.Forms.DependencyService.Get<ISaveWindowsPhone>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+else
+	Xamarin.Forms.DependencyService.Get<ISave>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+{% endhighlight %}
+
+{% endtabs %}  
+
+### Match case
+
+You can replace the text by matching case also.
```
[the remark this answers is not in the document]
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said @Ramaraj-Marimuthu sir

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 4]
```diff
@@ -0,0 +1,937 @@
...
+	Xamarin.Forms.DependencyService.Get<ISaveWindowsPhone>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+else
+	Xamarin.Forms.DependencyService.Get<ISave>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+{% endhighlight %}
+
+{% endtabs %}  
+
+### Whole words only
+
+You can replace the text by matching whole word only.
```
[the remark this answers is not in the document]
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said @Ramaraj-Marimuthu sir

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 5]
```diff
@@ -0,0 +1,937 @@
...
+	Xamarin.Forms.DependencyService.Get<ISaveWindowsPhone>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+else
+	Xamarin.Forms.DependencyService.Get<ISave>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+{% endhighlight %}
+
+{% endtabs %}  
+
+### Find and replace the first occurrence
+
+You can find the text and replace only the first occurrence of the text.
```
[the remark this answers is not in the document]
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said @Ramaraj-Marimuthu sir

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 6]
```diff
@@ -0,0 +1,937 @@
...
+
+The following code example illustrates how to find a particular text and replace only the first occurrence of the text.
```
[the remark this answers is not in the document]
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said @Ramaraj-Marimuthu sir

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 7]
```diff
@@ -0,0 +1,937 @@
...
+	Xamarin.Forms.DependencyService.Get<ISaveWindowsPhone>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+else
+	Xamarin.Forms.DependencyService.Get<ISave>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+{% endhighlight %}
+
+{% endtabs %}  
+
+## Find and replace in specific slide
+
+You can find text in a Presentation document and replace it with other text in a specific slide.
```
[the remark this answers is not in the document]
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said @Ramaraj-Marimuthu sir

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 8]
```diff
@@ -0,0 +1,937 @@
...
+	Xamarin.Forms.DependencyService.Get<ISaveWindowsPhone>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+else
+	Xamarin.Forms.DependencyService.Get<ISave>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+{% endhighlight %}
+
+{% endtabs %}  
+
+## Find and highlight in PowerPoint presentation
+
+.NET PowerPoint library allows you to find the text in the PowerPoint presentation and then to manipulate it.
```
[the remark this answers is not in the document]
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said @Ramaraj-Marimuthu sir

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 9]
```diff
@@ -0,0 +1,937 @@
...
+	Xamarin.Forms.DependencyService.Get<ISaveWindowsPhone>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+else
+	Xamarin.Forms.DependencyService.Get<ISave>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+{% endhighlight %}
+
+{% endtabs %}  
+
+## Find and highlight in specific slide
+
+You can find text in a specific slide and highlight the found text in the specific slide.
```
[the remark this answers is not in the document]
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said @Ramaraj-Marimuthu sir

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 10]
```diff
@@ -0,0 +1,932 @@
...
+title: Presentation Working with Find and Replace | Syncfusion
+description: This section illustrates how to find a particular text and replace it with another text or part of the document
+platform: file-formats
+control: Presentation
+documentation: UG
+keywords: 
+---
+# Working with Find and Replace
+
+.NET PowerPoint library allows you to search a particular text you like to change and replace it with another text in the PowerPoint presentation.
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:11]: .NET PowerPoint library allows you to search for a particular text that you want to change and replace it with another text in the PowerPoint presentation.

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 11]
```diff
@@ -0,0 +1,932 @@
...
+
+* You can replace either the first occurrence or all the occurrences of the text with another text in all PowerPoint slides and elements such as shape, textbox, table, smartart, etc.
+* You can replace the text in specific slide, notes slide, master slide or layout slide.
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:14]: You can replace the text in a specific slide, notes slide, master slide, or layout slide.

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 12]
```diff
@@ -0,0 +1,932 @@
...
+* You can replace the text by matching case, whole word and all occurrences or first occurrence alone.
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:15]: You can replace the text by matching the case, whole word, and all occurrences or the first occurrence alone.

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 13]
```diff
@@ -0,0 +1,932 @@
...
+* You can also replace the content that spans across several lines.
+
+## Find and replace in PowerPoint presentation
+
+You can find text in a PowerPoint presentation and replace it with other text.
+
+The following code example illustrates how to find all occurences of a particular text and replace it with other text in a PowerPoint presentation.
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:22]: occurences---->occurrences

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 14]
```diff
@@ -0,0 +1,932 @@
...
+
+{% highlight c# tabtitle="C#" %}
+//Opens an existing presentation.
+using (IPresentation pptxDoc = Presentation.Open("Sample.pptx"))
+{
+	//Finds all the occurrences of a particular text in the PowerPoint presentation
+	ITextSelection[] textSelections = pptxDoc.FindAll("product", false, false);
+	foreach (ITextSelection textSelection in textSelections)
+	{
+		//Gets the found text as single text part
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:34]: Gets the found text as a single text part

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 15]
```diff
@@ -0,0 +1,932 @@
...
+		ITextPart textPart = textSelection.GetAsOneTextPart();
+		//Replace the text
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:36]: Replaces the text

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 16]
```diff
@@ -0,0 +1,932 @@
...
+{% endhighlight %}
+
+{% highlight c# tabtitle="UWP" %}
+//Instantiates the File Picker
+FileOpenPicker openPicker = new FileOpenPicker();
+openPicker.SuggestedStartLocation = PickerLocationId.Desktop;
+openPicker.FileTypeFilter.Add(".pptx");
+//Creates a storage file from FileOpenPicker
+StorageFile inputStorageFile = await openPicker.PickSingleFileAsync();
+//Loads or open an PowerPoint Presentation
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:67]: Loads or opens a PowerPoint Presentation

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 17]
```diff
@@ -0,0 +1,932 @@
...
+	//Finds all the occurrences of a particular text in the PowerPoint presentation
+	ITextSelection[] textSelections = pptxDoc.FindAll("product", false, false);
+	foreach (ITextSelection textSelection in textSelections)
+	{
+		//Gets the found text as single text part
+		ITextPart textPart = textSelection.GetAsOneTextPart();
+		//Replace the text
+		textPart.Text = "Service";
+	}
+	//Save the PowerPoint Presentation as stream
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:103]: Saves the PowerPoint Presentation as a stream

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 18]
```diff
@@ -0,0 +1,932 @@
...
+	FileStream outputStream = new FileStream("Output.pptx", FileMode.Create);
+	pptxDoc.Save(outputStream);
+	//Release all resources of the stream
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:106]: Releases all resources of the stream

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 19]
```diff
@@ -0,0 +1,932 @@
...
+	//Closes the Presentation instance
+	pptxDoc.Close();
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+//"App" is the class of Portable project.
+Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+Stream inputStream = assembly.GetManifestResourceStream("SampleBrowser.Presentation.Samples.Template.Sample.pptx");
+//Loads or open an PowerPoint Presentation
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:117]: Loads or opens an PowerPoint Presentation

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 20]
```diff
@@ -0,0 +1,932 @@
...
+	Xamarin.Forms.DependencyService.Get<ISaveWindowsPhone>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+else
+	Xamarin.Forms.DependencyService.Get<ISave>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+{% endhighlight %}
+
+{% endtabs %}  
+
+### Match case
+
+You can find and replace the text by matching case also.
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:146]: You can find and replace the text by matching the case also.

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 21]
```diff
@@ -0,0 +1,932 @@
...
+
+The following code example illustrates how to find all occurences of a particular text by matching its case.
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:148]: The following code example illustrates how to find all occurrences of a particular text by matching the case.

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 22]
```diff
@@ -0,0 +1,932 @@
...
+//Finds all the occurrences of a particular text in the PowerPoint presentation
+ITextSelection[] textSelections = pptxDoc.FindAll("product", false, false);
+foreach (ITextSelection textSelection in textSelections)
+{
+	//Gets the found text as single text part
+	ITextPart textPart = textSelection.GetAsOneTextPart();
+	//Replace the text
+	textPart.Text = "Service";
+}
+//Create new memory stream to save Presentation.
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:128]: Create a new memory stream to save Presentation.

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 23]
```diff
@@ -0,0 +1,932 @@
...
+MemoryStream stream = new MemoryStream();
+//Save Presentation in stream format.
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:130]: Saves Presentation in stream format.

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 24]
```diff
@@ -0,0 +1,932 @@
...
+pptxDoc.Save(stream);
+//Close the presentation
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:132]: Closes the presentation

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 25]
```diff
@@ -0,0 +1,932 @@
...
+	pptxDoc.Save("Output.pptx");
+}
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+'Opens an existing presentation.
+Using pptxDoc As IPresentation = Presentation.Open("Sample.pptx")
+	Dim matchCase = True
+	Dim wholeWord = False
+	'Finds all the occurrences of a particular text which matches the given casing
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:177]: Finds all the occurrences of a particular text which matches the given casing

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 26]
```diff
@@ -0,0 +1,932 @@
...
+	Dim textSelections As ITextSelection() = pptxDoc.FindAll("product", matchCase, wholeWord)
+	For Each textSelection As ITextSelection In textSelections
+		'Gets the found text as single text part
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:180]: Gets the found text as a single text part

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 27]
```diff
@@ -0,0 +1,932 @@
...
+		Dim textPart As ITextPart = textSelection.GetAsOneTextPart()
+		'Replace the text
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:182]: Replace the text

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 28]
```diff
@@ -0,0 +1,932 @@
...
+		textPart.Text = "Service"
+	Next
+	'Saves the Presentation
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:185]: Saves the Presentation

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 29]
```diff
@@ -0,0 +1,932 @@
...
+	{
+		//Gets the found text as single text part
+		ITextPart textPart = textSelection.GetAsOneTextPart();
+		//Replace the text
+		textPart.Text = "Service";
+	}
+	//Save the PowerPoint Presentation as stream
+	FileStream outputStream = new FileStream("Output.pptx", FileMode.Create);
+	pptxDoc.Save(outputStream);
+	//Release all resources of the stream
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:376]: Releases all resources of the stream

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 30]
```diff
@@ -0,0 +1,932 @@
...
+	Dim textPart As ITextPart = textSelection.GetAsOneTextPart()
+	'Replace the text
+	textPart.Text = "Service"
+	'Saves the Presentation	
+	pptxDoc.Save("Output.pptx")
+End Using
+{% endhighlight %}
+
+{% highlight c# tabtitle="UWP" %}
+//Instantiates the File Picker
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:454]: use same casing for FilePicker.

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 31]
```diff
@@ -0,0 +1,932 @@
...
+	Xamarin.Forms.DependencyService.Get<ISaveWindowsPhone>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+else
+	Xamarin.Forms.DependencyService.Get<ISave>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+{% endhighlight %}
+
+{% endtabs %}  
+
+## Find and replace in specific slide
+
+You can find all occurences of a text in specific PowerPoint slide (slide, notes slide, master slide or layout slide) and replace it with other text.
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:530]: You can find all occurrences of a text in a specific PowerPoint slide (slide, notes slide, master slide, or layout slide) and replace it with other text.

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 32]
```diff
@@ -0,0 +1,932 @@
...
+
+The following code example illustrates how to find all occurences of a particular text and replace with other text in a specific slide.
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:532]: The following code example illustrates how to find all occurrences of a particular text and replace them with other text in a specific slide.

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 33]
```diff
@@ -0,0 +1,932 @@
...
+	Xamarin.Forms.DependencyService.Get<ISaveWindowsPhone>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+else
+	Xamarin.Forms.DependencyService.Get<ISave>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+{% endhighlight %}
+
+{% endtabs %}  
+
+## Find and highlight in PowerPoint presentation
+
+.NET PowerPoint library allows you to find all the occurences of a text in the PowerPoint presentation and highlight it.
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:661]: .NET PowerPoint library allows you to find all the occurrences of a text in the PowerPoint presentation and highlight it.

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 34]
```diff
@@ -0,0 +1,932 @@
...
+
+The following code example illustrates how to find all the occurences of a particular text and highlight it.
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:663]: occurrences

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 35]
```diff
@@ -0,0 +1,932 @@
...
+			textPart.Font.HighlightColor = ColorObject.Yellow;
+		}
+	}
+	//Saves the Presentation	
+	pptxDoc.Save("Output.pptx");
+}
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+'Opens an existing presentation.
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:688]: Opens an existing presentation.

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 36]
```diff
@@ -0,0 +1,932 @@
...
+Using pptxDoc As IPresentation = Presentation.Open("Sample.pptx")
+	'Finds all the occurrences of a particular text in the PowerPoint presentation
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:690]: Finds all the occurrences of a particular text in the PowerPoint presentation

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 37]
```diff
@@ -0,0 +1,932 @@
...
+	Dim textSelections As ITextSelection() = pptxDoc.FindAll("product", False, False)
+	For Each textSelection As ITextSelection In textSelections
+		'Gets the found text containing text parts
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:693]: Gets the found text containing text parts

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 38]
```diff
@@ -0,0 +1,932 @@
...
+		For Each textPart As ITextPart In textSelection.GetTextParts()
+			'Sets highlight color
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:695]: Sets highlight color

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 39]
```diff
@@ -0,0 +1,932 @@
...
+			textPart.Font.HighlightColor = ColorObject.Yellow
+		Next
+	Next
+	'Saves the Presentation
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:699]: Saves the Presentation

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 40]
```diff
@@ -0,0 +1,932 @@
...
+//The operation in Save under Xamarin varies between Windows Phone, Android and iOS platforms. Please refer presentation/xamarin section for respective code samples.
+if (Device.OS == TargetPlatform.WinPhone || Device.OS == TargetPlatform.Windows)
+	Xamarin.Forms.DependencyService.Get<ISaveWindowsPhone>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+else
+	Xamarin.Forms.DependencyService.Get<ISave>().Save("Output.pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation", stream);
+{% endhighlight %}
+
+{% endtabs %}  
+
+## Find and highlight in specific slide
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:794]: Find and highlight in a specific slide

File: File-Formats/Presentation/Working-with-find-and-replace.md [hunk 41]
```diff
@@ -0,0 +1,932 @@
...
+
+You can find all the occurences of a text in specific PowerPoint slide (slide, notes slide, master slide or layout slide) and highlight the found text.
```
Comment (laksmipriyamurugesan) [inline File-Formats/Presentation/Working-with-find-and-replace.md:796]: You can find all the occurrences of a text in a specific PowerPoint slide (slide, notes slide, master slide, or layout slide) and highlight the found text.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 10 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you requested.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 11 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 13 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 14 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 15 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 16 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 17 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 18 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 19 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 20 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 21 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 22 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 23 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 24 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 25 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md:177]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 26 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 27 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 28 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md:185]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 29 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 30 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 31 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 32 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 33 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 34 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 35 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md:688]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 36 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md:690]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 37 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md:693]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 39 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md:699]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 40 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

File: File-Formats/Presentation/Working-with-find-and-replace.md (hunk 41 above)
Comment (AkashArul26) [reply File-Formats/Presentation/Working-with-find-and-replace.md]: Modified as you said.

**Review by MohanaselvamJothi — APPROVED** [review]

**Review by Ramaraj-Marimuthu — APPROVED** [review]
I don't have any concerns in this.

**Pull request merged**